public/app/features/explore/Explore.tsx

```diff
@@ -283,6 +285,36 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     return <NoData />;
   }
 
+  renderCustom(width: number) {
+    const { timeZone, queryResponse, absoluteRange } = this.props;
+
+    const groupedByPlugin = queryResponse?.customFrames.reduce((acc: Record<string, DataFrame[]>, cur) => {
+      // cur.meta?.preferredVisualisationPluginId could be undefined by the API, but it won't be if we
+      // got here as it is checked in decorators.ts
+      if (!acc[`${cur.meta?.preferredVisualisationPluginId}`]) {
+        acc[`${cur.meta?.preferredVisualisationPluginId}`] = [cur];
+      } else {
+        acc[`${cur.meta?.preferredVisualisationPluginId}`].push(cur);
+      }
+      return acc;
+    }, {});
```

Comment:
I think lodash's `groupBy` should give the same results and be a bit more readable:

```suggestion
    const groupedByPlugin = groupBy(queryResponse?.customFrames, 'meta.preferredVisualisationPluginId')
```

Reply:
I have changed this to use lodash.